samples/Store/Domain/Cart.fs

```diff
@@ -76,22 +76,22 @@ let interpret command (state: Fold.State) =
     let toEventContext (reqContext: Context)            = { requestId = reqContext.requestId; time = reqContext.time }: Events.ContextInfo
     let (|Context|) (context: Context)                 = toEventContext context
     let maybePropChanges c skuId = function
-        | None -> []
+        | None -> [||]
         | Some waived ->
-            if not (itemExistsWithDifferentWaiveStatus skuId waived) then []
-            else [ Events.ItemPropertiesChanged { context = c; skuId = skuId; waived = waived } ]
+            if not (itemExistsWithDifferentWaiveStatus skuId waived) then [||]
+            else [| Events.ItemPropertiesChanged { context = c; skuId = skuId; waived = waived } |]
     let maybeQuantityChanges c skuId quantity =
-        if not (itemExistsWithDifferentQuantity skuId quantity) then [] else
-        [ Events.ItemQuantityChanged { context = c; skuId = skuId; quantity = quantity } ]
+        if not (itemExistsWithDifferentQuantity skuId quantity) then [||] else
+        [| Events.ItemQuantityChanged { context = c; skuId = skuId; quantity = quantity } |]
     match command with
     // a request to set quantity of `0` represents a removal request
     | SyncItem (Context c, skuId, Some 0, _) ->
-        if itemExistsWithSkuId skuId then [ Events.ItemRemoved { context = c; skuId = skuId } ]
-        else []
+        if itemExistsWithSkuId skuId then [| Events.ItemRemoved { context = c; skuId = skuId } |]
```

Comment:
same stuff here, either individual cases of the match, or maybe the whole thing should
wrap whole thing as a array comprehension

otherwise the noise is a bit much with that forest of hard to type symbols